iothub/device/src/DeviceAuthenticationWithTokenRefresh.cs

```diff
@@ -63,5 +67,19 @@ public override IotHubConnectionStringBuilder Populate(IotHubConnectionStringBui
             iotHubConnectionStringBuilder.DeviceId = DeviceId;
             return iotHubConnectionStringBuilder;
         }
+
+        private static int SetSasTokenSuggestedTimeToLiveSeconds(int suggestedTimeToLiveSeconds)
+        {
+            return suggestedTimeToLiveSeconds == 0
```

Comment:
`<=`?

Reply:
The `<` scenario will get validated in the base class `AuthenticationWithTokenRefresh` and will generate an `ArgumentOutOfRangeException`.